src/plugins/storage.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
```

Comment:
I think it's a nice little folder for unique components. @YaManicKill has also been doing this (e.g. themes.js) and I did this with push.js in the other PR.

Would keep the main folder clean with a couple of 'global' files.